pom.xml

```diff
@@ -247,11 +258,6 @@
       <plugin>
         <groupId>org.jenkins-ci.tools</groupId>
         <artifactId>maven-hpi-plugin</artifactId>
-        <configuration>
-          <systemProperties>
-            <hudson.bundled.plugins>${basedir}/work/plugins/ldap.hpl</hudson.bundled.plugins>
```

Comment:
Added years ago in fe4a7145a3656229c7dd1b3fcadb5d2f10d24185 and does not seem necessary any more.